src/main.js

```diff
@@ -129,7 +129,7 @@ function appendCoversToSavedView() {
     let thisCover = savedCovers[i];
     if(savedCoverIDs.includes(thisCover.id)) {
       continue;
-    }
+    } 
```

Comment:
I don't remember doing this I'm not sure why this is here lol

Reply:
probably accident - can you remove this change from this pr pls?

Reply:
done!